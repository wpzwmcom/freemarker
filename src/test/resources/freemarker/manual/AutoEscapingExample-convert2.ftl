<#outputformat "HTML"><#assign htmlMO><p>Test</#assign></#outputformat>
<#outputformat "XML"><#assign xmlMO><p>Test</p></#assign></#outputformat>
<#outputformat "RTF"><#assign rtfMO>\par Test</#assign></#outputformat>
<#-- We assume that we have "undefined" output format here. -->
HTML: ${htmlMO}
XML:  ${xmlMO}
RTF:  ${rtfMO}