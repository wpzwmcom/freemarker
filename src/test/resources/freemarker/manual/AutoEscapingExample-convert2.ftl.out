HTML: <p>Test
XML:  <p>Test</p>
RTF:  \par Test